figures/views.py

```diff
@@ -392,6 +393,125 @@ def get_serializer_context(self):
         return context
 
 
+class CourseMonthlyMetricsViewSet(CommonAuthMixin, viewsets.ViewSet):
+    """
+
+    """
+    months_back = 6
```

Comment:
Optional: To code the intention to do it.

```suggestion
    # TODO: Make it a query parameter.
    months_back = 6
```

Reply:
We'll also have to either make a maximum "months back" or paginate the results after a certain size